web/src/popover_menus.js

```diff
@@ -872,20 +888,29 @@ export function initialize() {
 
                 // Only show send later options that are possible today.
                 const date = new Date();
+                const day = date.getDay(); // Starts with 0 for Sunday.
                 const hours = date.getHours();
                 let possible_send_later_today = {};
+                let possible_send_later_monday = {};
                 if (hours <= 8) {
                     possible_send_later_today = send_later_today;
                 } else if (hours <= 15) {
                     possible_send_later_today.today_four_pm = send_later_today.today_four_pm;
                 } else {
                     possible_send_later_today = false;
                 }
+                // Show send_later_monday options only on Fridays and Saturdays.
+                if (day >= 5) {
+                    possible_send_later_monday = send_later_monday;
+                } else {
+                    possible_send_later_monday = false;
+                }
```

Comment:
The block starting here would also be fantastic to extract as a function that has node tests -- basically given a `date` as an input (that'll make it easy to write tests), we can have it return an object that would be the parameters to pass to `render_send_later_modal` with what possibilities we've selected.